modules/gdscript/language_server/gdscript_language_protocol.h

```diff
@@ -48,16 +49,16 @@ class GDScriptLanguageProtocol : public JSONRPC {
 
 	static GDScriptLanguageProtocol *singleton;
 
-	HashMap<int, Ref<WebSocketPeer> > clients;
-	WebSocketServer *server;
-	int lastest_client_id;
+	Vector<Ref<StreamPeerTCP> > clients;
+	TCP_Server *server;
+	int latest_client_ind;
 
 	Ref<GDScriptTextDocument> text_document;
 	Ref<GDScriptWorkspace> workspace;
 
-	void on_data_received(int p_id);
-	void on_client_connected(int p_id, const String &p_protocal);
-	void on_client_disconnected(int p_id, bool p_was_clean_close);
+	void on_data_received(int c_ind, int bytes);
```

Comment:
Parameters should follow godot code style using `p_` prefix for **in** and `r_` prefix for **out**.